src/app/resources/resources.service.ts

```diff
@@ -128,4 +131,38 @@ export class ResourcesService {
     return mediaTypes.find((type) => mimeType.indexOf(type) > -1) || 'other';
   }
 
+  sendResourceNotification(resource) {
+    const createdDate = resource.createdDate;
+    const currentUser = this.userService.get();
+    const resourceDocs = [];
+    const resourceNotification = forkJoin([
```

Comment:
It would be preferred to return the `forkJoin` without any subscription so if we add calls to this function in different components the other components can control what happens in success and error cases.

As a note, you don't need to assign this to a value and then immediately call it.  In the future if you do that you should remove the assignment and just call the function.